tests/autoload.php

```diff
@@ -10,4 +10,22 @@
 $classLoader->addPsr4("IO\\", __DIR__ . '/../vendor/plentymarkets/plugin-io/src', true);
 $classLoader->add('Findologic\Tests', __DIR__);
 
-$classLoader->register();
\ No newline at end of file
+/**
+ * Add class instances that may be defined in tests. In production this is defined by Plentymarkets.
+ *
+ * @var array<string, object> $classInstances
+ */
+global $classInstances;
```

Comment:
Necessary to define `global $classInstances;` twice?

Reply:
Yes since they're in different scopes. Here I am defining it as global and initializing it as an array. In the method `pluginApp` I just say that I want to get the global variable. If I wouldn't define it, it may be null, which I want to avoid.